fmriprep/interfaces/reports.py

```diff
@@ -169,10 +169,9 @@ def _generate_segment(self):
         if counts:
             header = '\t\t<ul class="elem-desc">'
             footer = '\t\t</ul>'
+            s = '' if n_runs == 1 else 's'
```

Comment:
`n_runs` isn't defined outside the loop.

```suggestion
```

Reply:
You mean outside `if`? Yes, it's a local variable.

Reply:
No, it's defined in the comprehension: `lines = [... for task_id, n_runs in sorted(counts.items())]`

Reply:
Oh, of course. My wrong.